ibis/backends/tests/test_client.py

```diff
@@ -1750,3 +1753,53 @@ def test_cross_database_join(con_create_database, monkeypatch):
     con.drop_table(left_table)
     con.drop_table(right_table, database=dbname)
     con.drop_database(dbname)
+
+
+@pytest.mark.notyet(
+    ["druid"], raises=NotImplementedError, reason="doesn't support create_table"
+)
+@pytest.mark.notimpl(["clickhouse"], reason="create table isn't implemented")
+@pytest.mark.notyet(["pandas", "dask", "polars"], reason="Doesn't support insert")
+@pytest.mark.notimpl(
+    ["flink"], reason="Temp tables are implemented as views, which don't support insert"
+)
+def test_insert_into_table_missing_columns(con, monkeypatch):
+    monkeypatch.setattr(ibis.options, "default_backend", con)
```

Comment:
Let's simplify the test by using `sg.parse_one(sql_string, read="whatever dialect you want")`.

Reply:
Thank you for this guidance! That made things much tidier.